app/models/activity.rb

```diff
@@ -23,6 +23,7 @@ class Activity < ApplicationRecord
     :intended_beneficiaries_step,
     :flow_step,
     :aid_type,
+    :oda_eligibility,
```

Comment:
For consistency with the other values here, could we name this `oda_eligibility_step`? It helps us tell these validation events apart from the `form_state` values. (I just noticed `aid_type` should have `_step` appended as well.)

Reply:
Oops! I was meant to write `oda_eligibility_step`. I'll amend it. Shall we do a quick new card for the `aid_type` step?

Reply:
I'll just make a quick PR to do that after this one is merged. If I do this at the same time, it will conflict with this one.